pkg/nsx/services/compare.go

```diff
@@ -2,10 +2,11 @@ package services
 
```

Comment:
Put builder.go and compare.go into a securitypolicy subdir?

Reply:
If so,  I think we should move all the files in services to securitypolicy subdir ?